### ISSUE TITLE
Sardanaeditor Ctrl+s does not save & apply

### ISSUE THREAD

**Issue by reszelaz** [sardana-org/sardana#911]
Ctrl+s seems to only save the file. One needs to go to the menu and execute "Save & apply" action manually.

**Comment by cpascual** [comment]
I would not waste improving the sardana editor

 (full disclosure: I always hated that feature and I think it should be removed from Sardana and let each person use its editor and just use hooks for the upload and reload)

**Comment by reszelaz** [comment]
I agree with you, and I think that the community as well - see the [enhancements ranking](https://hackmd.io/@reszelaz/Bk53UScj_#/22/2) (it is the third from the bottom, and there are numerous answers confirming what you say - see the previous slide). On one of the next Sardana follow-up meeting we will discuss on how to proceed with this widget - there are [14 issues](https://github.com/sardana-org/sardana/labels/gui%3Asardanaeditor) referring to it.
